src/toil/statsAndLogging.py

```diff
@@ -231,17 +232,45 @@ def shutdown(self) -> None:
         # in addition to cleaning on exceptions, onError should clean if there are any failed jobs
 
 
-def set_log_level(level: str, set_logger: Optional[logging.Logger] = None) -> None:
+def set_log_level(level: str, set_logger: Optional[logging.Logger] = None, colored_logs: bool = False) -> None:
```

Comment:
Maybe turning the colors on and off should have its own method? It's not really part of setting the log level to also set the colors. Or maybe this method's name could change?